### PR TITLE
add social svg to project

### PR DESCRIPTION
**Pull request by benschac**
## What does this PR do and why?
fix: #1565 assets for reference: https://www.figma.com/file/XEBCjxletE7rDdbj3Mk8lV/JBX---NFT-Rewards-UI---July-2022?node-id=76%3A1583

Step 1: import svg files. Note: copy-pasta'ed svg widthToHeight and height to match svg component pattern. I'll come back and change the width height logic if it doesn't work for project headers

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

**Comment by benschac** [comment]
Current dependencies on/for this PR:
* main
  * **PR #2365** <a href="https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2365" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/jbx-protocol/juice-interface/2365?utm_source=stack-comment).

File: src/components/icons/WebIconProjectHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+    >
+      <circle cx="21" cy="21" r="20" fill="#EEEBE3" stroke="#E0DBD2" />
+      <path
+        d="M32 21C32 27.0751 27.0751 32 21 32M32 21C32 14.9249 27.0751 10 21 10M32 21H10M21 32C14.9249 32 10 27.0751 10 21M21 32C23.7514 28.9878 25.315 25.0788 25.4 21C25.315 16.9212 23.7514 13.0122 21 10M21 32C18.2486 28.9878 16.685 25.0788 16.6 21C16.685 16.9212 18.2486 13.0122 21 10M10 21C10 14.9249 14.9249 10 21 10"
+        stroke="#AAA49A"
+        stroke-width="2"
+        stroke-linecap="round"
+        stroke-linejoin="round"
```
Comment (benschac) [inline src/components/icons/WebIconProjectHeader.tsx:20]: Unknown property. Will address if it's an issue, come back and delete if not needed.

Comment (benschac) [reply src/components/icons/WebIconProjectHeader.tsx:20]: these svg attrs came from the figma export.

**Review by benschac — COMMENTED** [review]
comment

**Review by jmilldotdev — APPROVED** [review]

**Comment by tomquirk** [comment]
@benschac wondering if we can use CSS to achieve this design, and use the icons that already exist in the codebase?

**Comment by benschac** [comment]
@tomquirk -- I'll give it a go and close this PR if CSS solution is ideal. Good idea.

**Pull request closed without merging**